File: Sorts/quickSort.js
const quickSort = (arr) => {
  if (arr.length <= 1) return arr

  const pivot = arr.shift();
  let left = [];
  let right = [];
  left = arr.filter(val => val < pivot);
  right = arr.filter(val => val >= pivot);

  return quickSort(left).concat(pivot).concat(quickSort(right));
};

const sortMe = [5, -10, 20, 100, 0.1, 1, 20, 30, -100, 10000, 400, 20, 45, -74];

console.log(quickSort(sortMe));

File: Sorts/quick_sort.py
def quick_sort(arr):
    if ( len(arr) <= 0 ):
        return arr
    
    pivot = arr.pop(0)
    left = [elem for elem in arr if elem < pivot]
    right = [elem for elem in arr if elem >= pivot]

    return quick_sort(left) + [pivot] + quick_sort(right)


print(quick_sort([4, 100, 0, -13, 4, 12, 44, 10]))